docs/multichain/destination/CertificateVerifier.md

```diff
@@ -58,12 +57,12 @@ struct ECDSAOperatorInfo {
 }
 
 /**
- * @notice updates the operator table
+ * @notice updates the operatorSet with the operator table (ie. stake weights) and its configuration
  * @param operatorSet the operatorSet to update the operator table for
- * @param referenceTimestamp the timestamp at which the operatorInfos were sourced
+ * @param referenceTimestamp the timestamp at which the operatorInfos (ie. operator table) was sourced
  * @param operatorInfos the operatorInfos to update the operator table with
- * @param operatorSetConfig the configuration of the operatorSet
- * @dev Only callable by the `OperatorTableUpdater`
+ * @param operatorSetConfig the configuration of the operatorSet, which includes the owner and max staleness period
+ * @dev Only callable by the `OperatorTableUpdater`, the function on which is permissionless
```

Comment:
```suggestion
 * @dev Only callable by the `OperatorTableUpdater`, the function on which is permissionless
```
@dev This function can only be called by the OperatorTableUpdater contract, which is itself permissionless to call.”